view/adminhtml/templates/boltpay/button.phtml

```diff
@@ -3,10 +3,14 @@
  * @var \Bolt\Boltpay\Block\Form $block
  */
 $code = $block->escapeHtml($block->getMethodCode());
+$backofficePublishableKey = $block->getPublishableKeyBackOffice();
 $customerCreditCardInfos = $block->getCustomerCreditCardInfo();
 $isAdminReorderForLoggedInCustomerFeatureEnabled = $block->isAdminReorderForLoggedInCustomerFeatureEnabled();
 ?>
 
+<?php if (!$backofficePublishableKey): ?>
+    In order to use Bolt from admin, please set "Publishable Key - Back Office" in the config.
```

Comment:
Maybe add an instruction where config is?
Like Stores > Configuration > Sales > Payment methods > Bolt Pay

Reply:
will do